notifications/views/emailtemplate.py

```diff
@@ -52,6 +53,14 @@ class CycleEmailTemplateEdit(CycleEmailTemplateBase, generic.UpdateView):
     success_message = 'Reporting cycle notification edited successfully'
     context_object_name = 'template'
 
+    def get_object(self):
+        obj = get_object_or_404(CycleEmailTemplate,
+                                id=self.kwargs['pk'])
+        if obj.is_triggered is False:
```

Comment:
Not so pretty: try 
if obj.is_triggered:
    raise PermissionDenied 
return obj  # no need for else since it won't reach here if error raised